generators/entity-server/templates/src/main/java/package/common/search_template.ejs

```diff
@@ -28,6 +28,7 @@
     <% } if (pagination !== 'no') { %>
     public ResponseEntity<List<<%= instanceType %>>> search<%= entityClassPlural %>(@RequestParam String query, Pageable pageable, HttpServletRequest request) {
         log.debug("REST request to search for a page of <%= entityClassPlural %> for query {}", query);<% if (viaService) { %>
+        pageable = Optional.ofNullable(pageable).orElse(Pageable.unpaged());
```

Comment:
is this related to this PR?

Reply:
No, it isn't. Sorry, maybe my own modification for other purposes, and left there.
I rollbacked it.